sys-devel/gettext/gettext-0.21.1-r1.ebuild

```diff
@@ -0,0 +1,178 @@
+	/usr/include/textstyle.h
+	/usr/include/textstyle/stdbool.h
+	/usr/include/textstyle/version.h
+	/usr/include/textstyle/woe32dll.h
+)
+
+PATCHES=(
+	"${FILESDIR}"/${PN}-0.19.7-disable-libintl.patch #564168
+	"${FILESDIR}"/${PN}-0.20-parallel_install.patch #685530
+	"${FILESDIR}"/${PN}-0.21_rc1-avoid_eautomake.patch
+	"${FILESDIR}"/${PN}-0.21-CVE-2020-12825.patch
+	"${FILESDIR}"/${P}-java-autoconf.patch
+)
+
+QA_SONAME_NO_SYMLINK=".*/preloadable_libintl.so"
+
+pkg_setup() {
+	java-pkg-opt-2_pkg_setup
+}
+
+src_prepare() {
+	java-pkg-opt-2_src_prepare
+
+	default
+
+	# gettext-0.21.1-java-autoconf.patch changes
+	# gettext-{runtime,tools}/configure.ac and the corresponding
+	# configure scripts. Avoid regenerating other autotools output.
+	touch -c gettext-{runtime,tools}/{aclocal.m4,Makefile.in,config.h.in,configure} || die
+
+	# Makefile.am adds a dependency on gettext-{runtime,tools}/configure.ac
+	touch -c configure || die
+
+	elibtoolize
+
+	use elibc_musl && eapply "${FILESDIR}"/${PN}-0.21-musl-omit_setlocale_lock.patch
+
+	rm gettext-tools/gnulib-lib/javaversion.class gettext-tools/m4/csharpexec-test.exe || die
+	sed -i 's;javaversion.class;;g' gettext-tools/gnulib-lib/Makefile.in || die
+	sed -i 's;csharpexec-test.exe;;g' gettext-tools/m4/Makefile.in gettext-tools/m4/Makefile.am || die
+	sed -i 's;^.*\.class;;g' gettext-tools/examples/Makefile.in gettext-tools/examples/Makefile.am || die
+}
+
+multilib_src_configure() {
+	local myconf=(
+		# switches common to runtime and top-level
+		--cache-file="${BUILD_DIR}"/config.cache
+		#--docdir="\$(datarootdir)/doc/${PF}"
+
+		# Emacs support is now in a separate package
+		--without-emacs
+		--without-lispdir
+		# glib depends on us so avoid circular deps
+		--with-included-glib
+		# libcroco depends on glib which ... ^^^
+		--with-included-libcroco
+		# this will _disable_ libunistring (since it is not bundled),
+		# see bug #326477
+		--with-included-libunistring
+		# Never build libintl since it's in dev-libs/libintl now.
+		--without-included-gettext
+		# Never build bundled copy of libxml2.
+		--without-included-libxml
+
+		--disable-csharp
+
+		$(use_enable acl)
+		$(use_enable cxx c++)
+		$(use_enable cxx libasprintf)
+		$(use_with git)
+		$(usex git --without-cvs $(use_with cvs))
+		$(multilib_native_use_enable java)
+		$(use_enable ncurses curses)
+		$(use_enable nls)
+		$(use_enable openmp)
+		$(use_enable static-libs static)
+	)
+
+	local ECONF_SOURCE="${S}"
+	if ! multilib_is_native_abi ; then
+		# for non-native ABIs, we build runtime only
+		ECONF_SOURCE+=/gettext-runtime
+	fi
+
+	econf "${myconf[@]}"
+}
+
+multilib_src_install() {
+	emake DESTDIR="${D}" install
+
+	if multilib_is_native_abi ; then
+		dosym msgfmt /usr/bin/gmsgfmt #43435
+		dobin gettext-tools/misc/gettextize
+	fi
+}
+
+multilib_src_install_all() {
+	find "${ED}" -type f -name "*.la" -delete || die
+
+	if use java ; then
+		java-pkg_dojar "${ED}"/usr/share/${PN}/*.jar
+		rm "${ED}"/usr/share/${PN}/*.jar || die
```

Comment:
How could other packages like [`net-vpn/i2p`](https://github.com/gentoo/gentoo/pull/29290) depend on it if it gets removed? Am I misreading this PR's intention?